src/components/Navigation.js

```diff
@@ -1,18 +1,24 @@
 import React from 'react';
-
+import {Link} from "react-router-dom"
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
+          <Link to="/" >
           <a href="">Home</a>
```

Comment:
You don't need to nest a tags here. Links act like a tags internally.
```suggestion
        Home
```